kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/InternalUtils.java

```diff
@@ -30,6 +33,18 @@
 public class InternalUtils {
     private InternalUtils() {}
 
+    private static final LocalDate EPOCH = LocalDate.ofEpochDay(0);
+
+    /**
+     * Utility method to get the number of days since epoch this given date is.
+     *
+     * @param date
+     */
+    public static int daysSinceEpoch(Date date) {
```

Comment:
Remove this from the DefaultKernelUtils if we have it here? I think it's in there if I remember correctly